sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/EndToEndTimeOutWithAvailabilityTest.java

```diff
@@ -176,6 +170,15 @@ public void testThresholdAvailabilityStrategyForReadsDefaultEnablementWithPpaf(
 
         for (int i = 0; i < maxIterations; i++) {
 
+            System.setProperty("COSMOS.IS_PER_PARTITION_AUTOMATIC_FAILOVER_ENABLED", "true");
```

Comment:
finally block should clean system property?

Reply:
Yeah, these are being cleared in a test-local finally block. Also, clearing newer properties in the `AfterClass` block too.